props/configs/src/main/resources/dh-defaults.prop

```diff
@@ -62,9 +62,9 @@ web.webgl.editable=true
 authentication.client.configuration.list=AuthHandlers
 
 # List of configuration properties to provide to authenticated clients, so they can interact with the server.
-client.configuration.list=java.version,deephaven.version,barrage.version,http.session.durationMs,file.separator,web.storage.layout.directory,web.storage.notebook.directory,web.webgl,web.webgl.editable
+client.configuration.list=java.version,deephaven.version,barrage.version,flight.version,http.session.durationMs,file.separator,web.storage.layout.directory,web.storage.notebook.directory,web.webgl,web.webgl.editable
 
 # Version list to add to the configuration property list. Each `=`-delimited pair denotes a short name for a versioned
 # jar, and a class that is found in that jar. Any such keys will be made available to the client.configuration.list
 # as <key>.version.
-client.version.list=deephaven=io.deephaven.engine.table.Table,barrage=io.deephaven.barrage.flatbuf.BarrageMessageWrapper
+client.version.list=deephaven=io.deephaven.engine.table.Table,barrage=io.deephaven.barrage.flatbuf.BarrageMessageWrapper,flight=org.apache.arrow.flight.impl.Flight
```

Comment:
Why do clients want to know the server's flight version? The server won't use the classes in `org.apache.arrow...` to send/receive anything to them, right?

Reply:
As a downstream user can easily add this themselves, I think this should be just the versions that will impact client-server communication. The c++ client for example intends (or intended to?) fail/warn if they detected a version on the server that the client wasn't using, and given the ... handling of breaking changes in flight format, I don't think we want to suggest there are incompatibilities where none are possible.

I'd vote remove this without a strong reason to have it.